temoa-uiuc/temoa-uiuc-chws.md

```diff
@@ -0,0 +1,49 @@
+# TEMOA's model of the CHWS
+
+* ExistingCapacity [Done]
+* Efficiency [Done]
+* CostInvest: No Invest Cost because UofI already has it
```

Comment:
There should be a ``costInvest``, which would be for deploying future Chilled Water Capacity.